bylaws.md

```diff
@@ -470,6 +470,12 @@ than the election of Cohort A directors. Officer Directors can also be
 removed subject to a vote of no confidence by the at-large Directors.
 In all other respects Officer Directors shall be as at-large Directors.
 
+**Section 5.5.2 Term Limits for Directors.** 
+No director shall serve more than six consecutive years. If a director
+has served for six years, that director shall become eligible again serve 
```

Comment:
`that director shall become eligible again to serve`

Reply:
Fixed.